src/scales/scale.logarithmic.js

```diff
@@ -134,8 +142,16 @@ module.exports = function(Chart) {
 			while (tickVal < me.max) {
 				ticks.push(tickVal);
 
-				var exp = Math.floor(helpers.log10(tickVal));
-				var significand = Math.floor(tickVal / Math.pow(10, exp)) + 1;
+                var exp;
+                var significand;
+
+                if(tickVal === 0){
+                    exp = Math.floor(helpers.log10(me.minNotZero));
+                    significand = Math.floor(me.minNotZero / Math.pow(10, exp));
```

Comment:
It's not an error - if we have min value > 0 10-5, we should have the first tick on 10-5, not 10-4.
I also improve accuracy in this calculations in the last commit.